documentation/docs/10-getting-started/10-introduction.md

```diff
@@ -4,7 +4,9 @@ title: Introduction
 
 ## Before we begin
 
-> SvelteKit is in release candidate phase for 1.0 while we address reported issues and add polish. If you get stuck, reach out for help in the [Discord chatroom](https://svelte.dev/chat).
+> If you're new to SvelteKit, do the [interactive tutorial](https://learn.svelte.dev/tutorial/introducing-sveltekit).
```

Comment:
```suggestion
> If you're new to SvelteKit, try the [interactive tutorial](https://learn.svelte.dev/tutorial/introducing-sveltekit).
```

Reply:
Why `try` over `do`?

Reply:
"do" reads slightly awkwardly to me. I'd probably make it something like "complete" if I wanted to make it a command. But it also feels slightly commanding to me whereas "try" is more of a recommendation than a command. It's pretty subtle and probably different people will have different views of it

Reply:
👍 I'll adjust it